yolov5/yolov5_trt.py

```diff
@@ -397,8 +402,8 @@ def run(self):
 
 if __name__ == "__main__":
     # load custom plugin and engine
-    PLUGIN_LIBRARY = "build/libmyplugins.so"
-    engine_file_path = "build/yolov5s.engine"
```

Comment:
Can you remove these changes?

Reply:
Sorry, I uploaded the pytorch related codes uncarefully. These codes have been corrected in my tensorrtx-phy branch.